change/@azure-msal-node-16f35a40-8506-4ab8-a9ed-cc62c026a5e2.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "patch",
+  "comment": {
+    "title": "",
```

Comment:
Would be nice to have a title for more descriptive change

Reply:
Our usual format is "<<desc>>  #PR"

Reply:
```suggestion
    "title": "Updated the regex and added a test for detecting public cert for SNI #4790",
```